docs/deployment/infrastructure.md

```diff
@@ -16,22 +16,31 @@ flowchart LR
     recaptcha[Google reCAPTCHA]
     rider((User's browser))
     idg[Identity Gateway]
-    mst_elig[MST Courtesy Card Eligibility Server]
+    mst_elig[Eligibility Server]
+    cc_data[(Card data)]
```

Comment:
```suggestion
    cc_data[(Courtesy Card data)]
```

I'm thinking we should add "Courtesy" in front since someone might think of payment cards when they see "Card"